src/client.ts

```diff
@@ -133,10 +132,10 @@ export class C8Client extends Fabric {
 
   async insertDocumentFromFile(
     collectionName: string,
-    csvPath: string,
+    file: any,
```

Comment:
Please avoid using `any`. Provide concrete types

Reply:
We suppose to use the `File | string` type instead of any. But passing data from a web browser is a File object whereas for node it is Csvstring. that's why I have added any type. 
Please let me if you any suggestions

Reply:
For time being, changed `any` type to `File | string`. In the future, if webpack provides a graceful  solution we will update it